sklearn/neighbors/tests/test_lof.py

```diff
@@ -44,18 +48,21 @@ def test_lof():
 
     # Assert predict() works:
     clf = neighbors.LocalOutlierFactor(contamination=0.25, n_neighbors=5).fit(X)
-    assert_array_equal(clf._predict(), 6 * [1] + 2 * [-1])
-    assert_array_equal(clf.fit_predict(X), 6 * [1] + 2 * [-1])
+    expected_predictions = 6 * [1] + 2 * [-1]
+    assert_array_equal(clf._predict(), expected_predictions)
+    assert_array_equal(clf.fit_predict(X), expected_predictions)
```

Comment:
Does `negative_outlier_factor_` been of the same `dtype` that `X`. If this is the case, we should either add a test or add a TODO.